tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ZepplinResource.java

```diff
@@ -71,4 +79,32 @@ public Response headZepplin() {
     public Response getZepplin() {
         return Response.ok().build();
     }
+
+    @POST
+    @APIResponse(responseCode = "200", description = "Review posted")
+    @APIResponse(responseCode = "404", description = "Review not found")
+    @Operation(summary = "Post by Zepplin", operationId = "postZepplin")
+    @Produces("text/plain")
+    @SecurityRequirement(name = "mutualTLSScheme", scopes = "zepplinScope")
+    @Consumes("multipart/form-data")
+    public Response postZepplin(
+            @RequestBody(description = "Record of a new user to be created in the system.", required = true,
+                         content = @Content(mediaType = "multipart/form-data",
+                                            schema = @Schema(properties = {
+                                                    @SchemaProperty(name = "passenger", description = "the passanger"),
+                                                    @SchemaProperty(name = "allergies",
+                                                                    description = "Does the passenger have any allergies"),
+                                                    @SchemaProperty(name = "specialRequests",
+                                                                    description = "Does the passenger have any special requests")
```

Comment:
Looks better, can we add set the `type` on each property? `allergies` and `specialRequests` should be `OBJECT`, `passenger` could be whatever.